wp-content/themes/core/assets/css/src/theme/spacing/aspect-ratios.pcss

```diff
@@ -0,0 +1,33 @@
+/* -------------------------------------------------------------------------
+ *
+ * Aspect Ratio Helpers
+ *
+ * ------------------------------------------------------------------------- */
+
+.s-aspect-21-9 {
```

Comment:
I've added these AR helpers under the spacing namespace as it seemed like the most relevant application.